unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderEntity/DCLBuilderInWorldEntity.cs

```diff
@@ -80,7 +79,7 @@ public bool IsVisible
 
     Material editMaterial;
 
-    Dictionary<string, GameObject> collidersDictionary = new Dictionary<string, GameObject>();
+    Dictionary<string, List<GameObject>> collidersDictionary = new Dictionary<string, List<GameObject>>();
```

Comment:
Renamed it, with the amounts of reworks that this piece of code has received, I found it more useful to work with GameObject, since a GameObject can have more than 1 colliders.